reinforcement_learning/rlclientlib/async_batcher.cc

```diff
@@ -0,0 +1,11 @@
+#include "logger/async_batcher.h"
+
+namespace reinforcement_learning {
+    queue_mode_enum to_queue_mode_enum(const char* queue_mode_str) {
```

Comment:
probably just 'queue_mode' is better:
1) it is consistent with other types / enums naming (see include/action_flags.h)
2) it is confusing to have queue_mode_enum as typename and queue_mode_str as variable name in a single scope.